tools/auto_tuner/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ list(APPEND CMAKE_MODULE_PATH
   ${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/
   ${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Modules
 )
-set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations -Wno-deprecated-declarations -Wno-shorten-64-to-32 -Wno-cast-align") 
```

Comment:
What still is marked as deprecated? is `-Wno-deprecated-declarations` still needed?

Reply:
I am keeping it also in the auto_tuner for consistency with the main project. Once we can remove it from portBLAS I'll remove it and test also in the auto_tuner. I say the flag was added twice here so I removed on occurrence